src/data/java_profile.rs

```diff
@@ -211,6 +212,19 @@ impl CollectData for JavaProfileRaw {
         }
 
         let data_dir = PathBuf::from(params.data_dir.clone());
+        for key in self.process_map.keys() {
+            let mut html_path = data_dir.clone();
+            html_path.push(format!("{}-java-flamegraph-{}.html", params.run_name, key));
+
+            let html_loc = html_path.to_str().unwrap();
+            let tmp_loc = format!(
+                "/tmp/aperf_tmp/{}-java-flamegraph-{}.html",
```

Comment:
We are hard-coding /tmp/aperf_tmp in multiple locations. We should have the tmp location passed in as a parameter.